### PR TITLE
Handle single file components

### PR DESCRIPTION
**Pull request by backflip**
Demo: https://github.com/jensfranke/feller/pull/2

Turned out to be a bit more complex than anticipated, unlike https://github.com/lixinliang/require-extension-vue/blob/master/index.js we need a transpile step with babel first (assuming we want to write ES2015 modules).

### CODE REVIEW

File: src/adapter.js [hunk 1]
```diff
@@ -70,14 +113,17 @@ class VueAdapter extends Adapter {
 	updateVueComponent(view) {
 		const component = this._source.find(view.handle);
 
+		// TODO: Figure out how to pass path. view.handle?
```
Comment (backflip) [inline src/adapter.js:116]: I wasn't able to test this part. How would I trigger this?

Comment (swey) [reply src/adapter.js:116]: With changes in a nested component - but looks good

**Comment by swey** [comment]
I based on your approach and made some refactoring and started to add the support for explicitly defined `props`: https://github.com/swey/fractal-vue-adapter/tree/feature/sfc

But still one problem to solve

**Comment by swey** [comment]
Also see #2

**Pull request closed without merging**